### PR TITLE
Rename item fragment name to title

### LINKED ISSUE #163: Item name should move to item title
**Is this a BUG REPORT or FEATURE REQUEST?**:
bug

**What happened**:
Currently a single item for the items fragment is using `name`.

**What you expected to happen**:
Do be more consistent we should switch to `title`.

### PR DESCRIPTION
**Pull request by lauragift21**
**What this PR does / why we need it**: 
Change name to title within the item fragment

**Which issue this PR fixes** *(optional - uncomment and add issue)*:
fixes #163

### CODE REVIEW

**Comment by stp-ip** [comment]
This also needs a change in the layout for the items fragment.

**Comment by lauragift21** [comment]
oh yeah i missed that.

File: layouts/partials/fragments/items.html [hunk 1]
```diff
@@ -64,9 +64,9 @@ <h5>
                   {{- if eq $bg "primary" -}}
                     {{- printf " text-%s" "dark" -}}
                   {{- end -}}
-                ">{{ $items.name | markdownify }}</a>
+                ">{{ $items.title | markdownify }}</a>
```
Comment (stp-ip) [inline layouts/partials/fragments/items.html]: `.Title`

File: layouts/partials/fragments/items.html [hunk 2]
```diff
@@ -64,9 +64,9 @@ <h5>
...
               {{- else }}
-                {{- $items.name | markdownify -}}
+                {{- $items.title | markdownify -}}
```
Comment (stp-ip) [inline layouts/partials/fragments/items.html]: `.Title`

**Comment by lauragift21** [comment]
Alright. I'd change that. No one questions Hugo ways.

**Pull request merged**